CorpusSearch/newdocs

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#note: this script needs to be run on the manx-search-data repo
+# not the repo it lives in
+# weirdest trick is the pattern matching to get dir from fdir
+# basically ## means delete the longest matching pattern from the var, # the shortest
+# for more on that see https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
+
+MAX=20
+SINCE=three-months-ago
+DIR_ONLY=0
+usage="$(basename "$0") [-d] [-h] [-m MAX] [-s SINCE]  - list recently changed docs when run in manx-search-data
+
+options are:
+    -d  print the last directory only
```

Comment:
I don't understand what this does, or the reason it's necessary. Can it be removed?

Reply:
It's much easier for me to read that output to see what changed than read the whole path

Reply:
`-d`

Reply:
Again, just so much easier to read that output

Reply:
Done